src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.cs

```diff
@@ -127,4 +127,12 @@ internal class SafeSharedAsn1ObjectHandle : SafeInteriorHandle
         {
         }
     }
+
+    internal class SafeSharedAsn1OctetStringHandle: SafeInteriorHandle
```

Comment:
The name of the preceding type is a bit confusing to someone not immersed in the material.  `Asn1Object` doesn't mean "the base type of all ASN.1 structured data", it means "the ASN.1 `OBJECT IDENTIFIER` type", which we call Oid.

But, since libcrypto calls it `ASN1_OBJECT`, I made the name here match the name there.